### PR TITLE
[IT] Added `Client/File_Formats/*`

### PR DESCRIPTION
**Pull request by massimoty**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [ ] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/File_formats/osr_(file_format)/it.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+| ULEB128 | Variabile | Un intero di lunghezza variabile. Vedere [ULEB128](https://en.wikipedia.org/wiki/LEB128). |
+| Stringa | Variabile | Ha tre parti: un singolo byte che può essere 0x00, che indica che le due parti successive non sono presenti, oppure 0x0b (11 decimale), che indica che le due parti successive sono presenti. Se è 0x0b, ci sarà un ULEB128, che rappresenta la lunghezza in byte della stringa seguente, e poi la stringa stessa, codificata in UTF-8. Vedere [UTF-8](https://it.wikipedia.org/wiki/UTF-8) |
+
+## Formato
+
+Gli offset dei byte non sono inclusi in questa tabella a causa dei valori di lunghezza variabile.
+
+| Tipo di dati | Descrizione |
+| :-- | :-- |
+| Byte | Modalità di gioco del replay (0 = osu!Standard, 1 = Taiko, 2 = Catch the Beat, 3 = osu!mania).
```
Comment (Sclafus) [inline wiki/Client/File_formats/osr_(file_format)/it.md:22]:
```suggestion
| Byte | Modalità di gioco del replay (0 = osu!Standard, 1 = Taiko, 2 = Catch the Beat, 3 = osu!mania). |
```

File: wiki/Client/File_formats/osr_(file_format)/it.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+| Integer | Versione del gioco al momento della creazione del replay (es. 20131216) |
+| Stringa | hash MD5 di osu! beatmap |
+| Stringa | Nome del giocatore |
+| Stringa | hash MD5 di osu! replay (include alcune proprietà del replay) |
+| Breve | Numero di 300s |
+| Short | Numero di 100 in standard, 150 in Taiko, 100 in CTB, 100 in mania |
+| Breve | Numero di 50s in standard, frutti piccoli in CTB, 50s in mania |
+| Breve | Numero di Gekis in standard, Max 300s in mania |
+| Breve | Numero di Katus in standard, 200 in mania |
+| Breve | Numero di missioni
```
Comment (Sclafus) [inline wiki/Client/File_formats/osr_(file_format)/it.md:32]:
```suggestion
| Breve | Numero di missioni | 
```

File: wiki/Client/File_formats/osr_(file_format)/it.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+| Integer | Punteggio totale visualizzato nel report dei punteggi |
+| Breve | Combo più grande visualizzata sul report del punteggio |
+| Byte | Combo perfetta/piena (1 = nessun errore, nessuna interruzione del cursore e nessun cursore finito in anticipo).
```
Comment (Sclafus) [inline wiki/Client/File_formats/osr_(file_format)/it.md:35]:
```suggestion
| Byte | Combo perfetta/piena (1 = nessun errore, nessuna interruzione del cursore e nessun cursore finito in anticipo). |
```

File: wiki/Client/File_formats/osr_(file_format)/it.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+| Integro | Mods utilizzati. Vedere di seguito l'elenco dei valori dei moduli. |
+| Stringa | Grafico a barre della vita: coppie separate da virgole u/v, dove u è il tempo in millisecondi della canzone e v è un valore in virgola mobile da 0 a 1 che rappresenta la quantità di vita che si ha in quel momento (0 = la barra della vita è vuota, 1= la barra della vita è piena).
```
Comment (Sclafus) [inline wiki/Client/File_formats/osr_(file_format)/it.md:37]:
```suggestion
| Stringa | Grafico a barre della vita: coppie separate da virgole u/v, dove u è il tempo in millisecondi della canzone e v è un valore in virgola mobile da 0 a 1 che rappresenta la quantità di vita che si ha in quel momento (0 = la barra della vita è vuota, 1= la barra della vita è piena). |
```

File: wiki/Client/File_formats/osr_(file_format)/it.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+| Lungo | Stampa dell'ora con ([Windows ticks](https://learn.microsoft.com/it-it/dotnet/api/system.datetime.ticks)) |
+| Integer | Lunghezza in byte dei dati di replay compressi |
+| Array di Byte | Dati di replay compressi |
+| Lungo | ID punteggio online |
+| Double | Informazioni aggiuntive sul mod. Presente solo se [Target Practice](/wiki/Gameplay/Game_modifier/Target_Practice) è abilitato. |
+
+
```
Comment (Sclafus) [inline wiki/Client/File_formats/osr_(file_format)/it.md:44]:
```suggestion
```

File: wiki/Client/File_formats/osr_(file_format)/it.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+**Informazioni Aggiuntive Sulle Mod**
+
+| Mod | Informazioni memorizzate
```
Comment (Sclafus) [inline wiki/Client/File_formats/osr_(file_format)/it.md:47]:
```suggestion
| Mod | Informazioni memorizzate |
```

**Review by Sclafus — COMMENTED** [review]
Added the CI requested changes

**Review by githacki — CHANGES_REQUESTED** [review]
Fix [`remark-lint:table-pipes`](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-table-pipes#readme) warning

File: wiki/Client/File_formats/osr_(file_format)/it.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+| Integer | Versione del gioco al momento della creazione del replay (es. 20131216) |
+| Stringa | hash MD5 di osu! beatmap |
+| Stringa | Nome del giocatore |
+| Stringa | hash MD5 di osu! replay (include alcune proprietà del replay) |
+| Breve | Numero di 300s |
+| Short | Numero di 100 in standard, 150 in Taiko, 100 in CTB, 100 in mania |
+| Breve | Numero di 50s in standard, frutti piccoli in CTB, 50s in mania |
+| Breve | Numero di Gekis in standard, Max 300s in mania |
+| Breve | Numero di Katus in standard, 200 in mania |
+| Breve | Numero di missioni |
```
Comment (githacki) [inline wiki/Client/File_formats/osr_(file_format)/it.md:32]:
```suggestion
| Breve | Numero di missioni |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 8]
```diff
@@ -0,0 +1,363 @@
...
+
+**`.osu`** è un formato di file leggibile dall'uomo contenente informazioni su una beatmap.
+
+## Struttura
+
+La prima riga del file specifica la versione del formato del file. Ad esempio, `osu file format v14` è la versione più recente.
+
+Il contenuto seguente è separato in sezioni, indicate dai titoli delle sezioni tra parentesi quadre.
+
+| Sezione | Descrizione | Tipo di contenuto
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:11]:
```suggestion
| Sezione | Descrizione | Tipo di contenuto |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 9]
```diff
@@ -0,0 +1,363 @@
...
+| `[Eventi]` | Eventi grafici di beatmap e storyboard | Elenchi separati da virgole |
+| `[TimingPoints]` | Punti di cronometraggio e controllo | Elenchi separati da virgole |
+| `[Colours]` | Colori della combo e della skin | Coppie `key : value` |
+| `[HitObjects]` Oggetti colpiti | Elenchi separati da virgole |
+
+## Generale
+
+<!-- TODO: this is missing some functional options that are leftover from very old file formats -->
+
+| Opzione | Tipo di valore | Descrizione | Valore predefinito
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:26]:
```suggestion
| Opzione | Tipo di valore | Descrizione | Valore predefinito |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 10]
```diff
@@ -0,0 +1,363 @@
...
+| :-- | :-- | :-- | :-- |
+| `AudioFilename` | Stringa | Posizione del file audio rispetto alla cartella corrente | |
+| `AudioLeadIn` | Integer | Millisecondi di silenzio prima dell'inizio della riproduzione dell'audio | 0 |
+| `AudioHash` | Stringa | *Deprecated* | |
+| `PreviewTime` | Integer | Tempo in millisecondi in cui deve iniziare l'anteprima audio | -1 |
+| `Countdown` | Integer | Velocità del conto alla rovescia prima del primo oggetto colpito (`0` = nessun conto alla rovescia, `1` = normale, `2` = metà, `3` = doppio) | 1
+| `SampleSet` | Stringa | Set di campioni che verrà utilizzato se i punti di temporizzazione non lo sovrascrivono (`Normal`, `Soft`, `Drum`) | Normal |
+| `StackLeniency` | Decimale | [Moltiplicatore](/wiki/Beatmap/Stack_leniency) per la soglia nel tempo in cui gli oggetti colpiti posti vicini si impilano (0-1) | 0,7 |
+| `Mode` | Integer | Game mode (`0` = osu!, `1` = osu!taiko, `2` = osu!catch, `3` = osu!mania) | 0
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:35]:
```suggestion
| `Mode` | Integer | Game mode (`0` = osu!, `1` = osu!taiko, `2` = osu!catch, `3` = osu!mania) | 0 |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 11]
```diff
@@ -0,0 +1,363 @@
...
+| `LetterboxInBreaks` | 0 o 1 | Se le interruzioni hanno o meno un effetto letterboxing | 0 |
+| `StoryFireInFront` | 0 o 1 | *Deprecated* | 1 |
+| `UseSkinSprites` | 0 o 1 | Se lo storyboard può usare o meno le immagini della skin dell'utente | 0 |
+| `AlwaysShowPlayfield` | 0 o 1 | *Deprecated* | 0 |
+| `OverlayPosition` | Stringa | Sorteggia l'ordine di sovrapposizione dei cerchi colpiti rispetto ai numeri colpiti (`NoChange` = usa l'impostazione della skin, `Below` = sorteggia le sovrapposizioni sotto i numeri, `Above` = sorteggia le sovrapposizioni sopra i numeri) | NoChange
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:40]:
```suggestion
| `OverlayPosition` | Stringa | Sorteggia l'ordine di sovrapposizione dei cerchi colpiti rispetto ai numeri colpiti (`NoChange` = usa l'impostazione della skin, `Below` = sorteggia le sovrapposizioni sotto i numeri, `Above` = sorteggia le sovrapposizioni sopra i numeri) | NoChange |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 12]
```diff
@@ -0,0 +1,363 @@
...
+| :-- | :-- | :-- |
+| `Bookmarks` | Elenco separato da virgole di numeri interi | Tempo in millisecondi dei [bookmarks](/wiki/Client/Beatmap_editor/Compose#bottom-(song's-timeline)) |
+| `DistanceSpacing` | Decimale | Moltiplicatore dei [Distance snap](/wiki/Client/Beatmap_editor/Distance_snap) |
+| `BeatDivisor` | Integer | [Beat snap divisor](/wiki/Client/Beatmap_editor/Beat_snap_divisor) |
+| `GridSize` | Integer | [Dimensione griglia](/wiki/Beatmapping/Grid_snapping) |
+| `TimelineZoom` | Decimale | Fattore di scala per la [timeline degli oggetti](/wiki/Client/Beatmap_editor/Compose#top-left-(hit-objects-timeline)) |
+
+## Metadata
+
+| Opzione | Tipo di valore | Descrizione
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:62]:
```suggestion
| Opzione | Tipo di valore | Descrizione |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 13]
```diff
@@ -0,0 +1,363 @@
...
+| `Creator` | Stringa | Creatore della beatmap |
+| `Versione` | Stringa | Nome della difficoltà |
+| `Source` | Stringa | Media originale per cui il brano è stato prodotto |
+| `Tags` | Elenco di stringhe separate dallo spazio | Termini di ricerca |
+| `BeatmapID` | Integer | ID della difficoltà |
+| `BeatmapSetID` | Integro | ID beatmap |
+
+## Difficulty
+
+| Opzione | Tipo di valore | Descrizione
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:77]:
```suggestion
| Opzione | Tipo di valore | Descrizione |
```

**Review by githacki — CHANGES_REQUESTED** [review]
Fix [`remark-lint:table-pipes`](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-table-pipes#readme) warning

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 14]
```diff
@@ -0,0 +1,363 @@
...
+Questo hitsound utilizza le proprietà `hitSound` e `hitSample` dell'oggetto hit, ma sono supportati solo i suoni normale e fischio. Il nome del file è `<sampleSet>-slider<hitSound><index>.wav`, dove `hitSound` è `slide` per i suoni normali o `whistle` per i suoni fischiati.
+
+### Spinner
+
+*Sintassi degli spinner:* `x,y,time,type,hitSound,endTime,hitSample`
+
+- **`endTime` (intero):** Tempo di fine dello spinner, in millisecondi dall'inizio dell'audio della beatmap.
+- `x` e `y` non influiscono sugli spinner. Sono predefiniti al centro del campo di gioco, `256,192`.
+
+### Trattenute (solo osu!mania)
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:298]:
```suggestion
### Note lunghe (solo osu!mania)
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 15]
```diff
@@ -0,0 +1,363 @@
...
+
+Il campo di gioco di osu!catch utilizza solo l'asse x, quindi `y` non è rilevante. Le curve dei cursori possono utilizzare lo spazio verticale per ottenere un'accelerazione orizzontale quando vengono appiattite in un campo di gioco monodimensionale.
+
+- I cerchi dei colpi diventano frutti.
+- I cursori diventano flussi di succo, con frutti su ogni bordo.
+- Gli spinner diventano docce di banane
+
+### osu!mania
+
+Analogamente a osu!catch, gli oggetti colpiti in osu!mania non usano `y`. Per determinare la colonna si usa `x`; si veda la sezione [Holds] (#holds-(osu!mania-only)).
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:360]:
```suggestion
Analogamente a osu!catch, gli oggetti colpiti in osu!mania non usano `y`. Per determinare la colonna si usa `x`; si veda la sezione [Note lunghe] (#note-lunghe-(solo-osu!mania)).
```

**Review by githacki — CHANGES_REQUESTED** [review]
Fix [`remark-lint:no-undefined-references`](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-no-undefined-references#readme) warning

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 16]
```diff
@@ -0,0 +1,363 @@
...
+
+Il campo di gioco di osu!catch utilizza solo l'asse x, quindi `y` non è rilevante. Le curve dei cursori possono utilizzare lo spazio verticale per ottenere un'accelerazione orizzontale quando vengono appiattite in un campo di gioco monodimensionale.
+
+- I cerchi dei colpi diventano frutti.
+- I cursori diventano flussi di succo, con frutti su ogni bordo.
+- Gli spinner diventano docce di banane
+
+### osu!mania
+
+Analogamente a osu!catch, gli oggetti colpiti in osu!mania non usano `y`. Per determinare la colonna si usa `x`; si veda la sezione [Note lunghe] (#note-lunghe-(solo-osu!mania)).
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:360]:
```suggestion
Analogamente a osu!catch, gli oggetti colpiti in osu!mania non usano `y`. Per determinare la colonna si usa `x`; si veda la sezione [Note lunghe](#note-lunghe-(solo-osu!mania)).
```

**Review by githacki — CHANGES_REQUESTED** [review]
Further CI error fix

**Comment by Sclafus** [comment]
Please fix it on your own @massimoty there's too much stuff that is wrong D:

**Review by githacki — CHANGES_REQUESTED** [review]
As per the [Italian osu! wiki contributors' glossary](https://github.com/osu-italy/osu-wiki#glossary):
- Hit samples -> Campioni audio
- Cursore -> Slider
- Keep data types untranslated
- Timing points -> Punti di tempo
- Timing section -> Sezione temporale
- Kiai time -> Sezione kiai

Please, continue the translation from line 158 on, being mindful of the aforementioned changes and the wording of the review comments sent so far.

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 17]
```diff
@@ -0,0 +1,363 @@
...
+
+### Tipi
+
+I tipi di oggetti da colpire sono memorizzati in un integer a 8 bit, dove ogni bit è un flag con un significato speciale. Il tipo di oggetto da colpire di base è dato dai bit 0, 1, 3 e 7 (dal meno significativo al più significativo):
+
+- 0: Cerchio
+- 1: Cursore
+- 3: Spinner
+- 7: osu!mania hold
+
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:198]:
```suggestion
| 2 | Segna l'inizio di una nuova combo |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 18]
```diff
@@ -0,0 +1,363 @@
...
+I bit rimanenti sono utilizzati per distinguere le nuove combo e per saltare i colori delle combo (comunemente chiamato "colour hax"):
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:199]:
```suggestion
| 3 | Segna l'oggetto come spinner  |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 19]
```diff
@@ -0,0 +1,363 @@
...
+
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:200]:
```suggestion
| 4, 5, 6 | Un integer a 3 bit che specifica quanti colori della combo saltare, una pratica alla quale ci si riferisce con "colour hax". Rilevante solo se l'oggetto inizia una nuova combo |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 20]
```diff
@@ -0,0 +1,363 @@
...
+- 2: Nuova combo
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:201]:
```suggestion
| 7 | Segna l'oggetto come nota lunga di osu!mania |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 21]
```diff
@@ -0,0 +1,363 @@
...
+- 4-6: un intero a 3 bit che specifica quanti colori della combo saltare, se questo oggetto inizia una nuova combo.
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:202]:
```suggestion
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 22]
```diff
@@ -0,0 +1,363 @@
...
+
+### Hitsounds
+
+I bit flag `hitSound` determinano quali suoni verranno riprodotti quando l'oggetto viene colpito:
+
+- 0: Normale
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:208]:
```suggestion
| 0 | Normal |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 23]
```diff
@@ -0,0 +1,363 @@
...
+- 1: Fischio
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:209]:
```suggestion
| 1 | Whistle |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 24]
```diff
@@ -0,0 +1,363 @@
...
+- 2: Fine
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:210]:
```suggestion
| 2 | Finish |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 25]
```diff
@@ -0,0 +1,363 @@
...
+- 3: Battito di mani
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:211]:
```suggestion
| 3 | Clap |
```

File: wiki/Client/File_formats/osu_(file_format)/it.md [hunk 26]
```diff
@@ -0,0 +1,363 @@
...
+
+I bit rimanenti sono utilizzati per distinguere le nuove combo e per saltare i colori delle combo (comunemente chiamato "colour hax"):
+
+- 2: Nuova combo
+- 4-6: un intero a 3 bit che specifica quanti colori della combo saltare, se questo oggetto inizia una nuova combo.
+
+### Hitsounds
+
+I bit flag `hitSound` determinano quali suoni verranno riprodotti quando l'oggetto viene colpito:
+
```
Comment (githacki) [inline wiki/Client/File_formats/osu_(file_format)/it.md:207]:
```suggestion

| Bit index | Meaning |
| :-- | :-- |
```

**Review by githacki — CHANGES_REQUESTED** [review]
Apply #9846

**Comment by TicClick** [comment]
it's been 2 months since the PR was open -- is there anything left to do to get it merged in?

**Comment by githacki** [comment]
> it's been 2 months since the PR was open -- is there anything left to do to get it merged in?

yes, there is, i had requested @massimoty to fix last 200 lines but he hasn't. atp i guess i'll just send a review for those last lines in these days and approval when it will be ready for merge

**Comment by Sclafus** [comment]
I think this may be closed, we will address it in another PR.

**Pull request closed without merging**